src/libraries/Microsoft.PowerFx.Core/Public/Values/DateTimeValue.cs

```diff
@@ -28,6 +28,12 @@ public override void Visit(IValueVisitor visitor)
 
         public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettings settings)
         {
+            if (settings.UseCompactRepresentation)
+            {
```

Comment:
Can you add a comment as to the difference. 
And maybe include an example output of both cases